common/workunit/workunit.hpp

```diff
@@ -152,7 +152,8 @@ enum WUAction
     WUActionPause = 5,
     WUActionPauseNow = 6,
     WUActionResume = 7,
-    WUActionSize = 8
+    WUActionSize = 8,
```

Comment:
WUActionSize should be last - and should have a comment to that effect..

Reply:
whoops. Fixed.